src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -512,7 +512,10 @@ private function get_currency_position( array $record ) {
 	 * @return string
 	 */
 	private function get_post_excerpt( $event_id, $import_excerpt ) {
-		_deprecated_function( __METHOD__, 5.1.6', '$this->get_post_text_field( $event_id, $record, "event_excerpt", "post_excerpt" )' );
+		_deprecated_function( __METHOD__,
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 989ab547c1202381d133e6eb49de0d33</sub>